reps_event_app/lib/ui/events.dart

```diff
@@ -60,6 +60,11 @@ class _EventsState extends State<Events> {
           ],
           bottom: TabBar(
             indicatorColor: Colors.white,
+            onTap: (kTabLabel) {
```

Comment:
The keyboard will hide when the user is switching the tab to About Section.  For a better UX can we hide the keyboard when the user is switching the tab from 0 to 1 or vice versa?
To do so, just remove the `KTabLabel == 2` and it should work fine.

Reply:
But still, if the user swipes to go to Reps tab then keyboard won't hide.

Reply:
Yes, the keyboard won't hide during the swipe. 
But, adding the check mentioned below in the `build` function won't allow users to open the keyboard. You may give it a try.
```
if (FocusScope.of(context).hasPrimaryFocus) {
      FocusScope.of(context).unfocus();
   }
```

Reply:
> 
> 
> I tried but as you said it wasn't allowing the keyboard to open.

Try this, create a `TabController _tabController` and in the `initState` method attach a listener to `_tabController` to listen to tab changes and in the listener function remove the focus.

Reply:
Ya, I even tried that but again it was working for the taps but not for swipe. Moreover, it was making the tab changes unsmooth.

Reply:
Okay, I'll merge this request and open a new issue for swipe.
Thank you for your contribution.